.github/actions/compute-matrix/action.yaml

```diff
@@ -33,12 +35,26 @@ runs:
           - { CUDA_VER: '12.0.1', ARCH: 'arm64', PY_VER: '3.11', LINUX_VER: 'ubuntu20.04', gpu: 'a100', driver: 'latest' }
         "
 
+        export TEST_MATRIX_CONDA="
+          - { CUDA_VER: '12.0.1', ARCH: 'amd64', PY_VER: '3.9', LINUX_VER: 'ubuntu20.04', gpu: 'v100', driver: 'latest' }
+          - { CUDA_VER: '12.0.1', ARCH: 'amd64', PY_VER: '3.10', LINUX_VER: 'ubuntu20.04', gpu: 'v100', driver: 'latest' }
+          - { CUDA_VER: '12.0.1', ARCH: 'amd64', PY_VER: '3.11', LINUX_VER: 'ubuntu20.04', gpu: 'v100', driver: 'latest' }
+          - { CUDA_VER: '12.0.1', ARCH: 'arm64', PY_VER: '3.9', LINUX_VER: 'ubuntu20.04', gpu: 'a100', driver: 'latest' }
+          - { CUDA_VER: '12.0.1', ARCH: 'arm64', PY_VER: '3.10', LINUX_VER: 'ubuntu20.04', gpu: 'a100', driver: 'latest' }
+          - { CUDA_VER: '12.0.1', ARCH: 'arm64', PY_VER: '3.11', LINUX_VER: 'ubuntu20.04', gpu: 'a100', driver: 'latest' }
```

Comment:
What's the rationale for having two separate test matrices? AFAICT the only difference is that on x86 wheels test on Ubuntu 18.04 while conda tests on 20.04. Why can't/shouldn't we unify them?

Reply:
I am okay with unifying them. I created a separate test matrix for conda because we can't test 18.04 with conda (no images for that) and didn't want to change the wheel test matrix.

Reply:
Gotcha. For the purpose of verifying that we're making manylinux wheels properly I'm fine testing on older systems at the cost of keeping two separate matrices.

Reply:
Unified the test matrices and dropped Ubuntu 18.04 in 5a27d41bb71f5cac728bb0a8d83e9d91ef267dc6.

Reply:
Also OK with me.